metrics/resetting_timer.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 )
 
-// Initial slice capacity for the values stored in a ResettingTimer
+// InitialResettingTimerSliceCap sets the tnitial slice capacity for the values stored in a ResettingTimer
```

Comment:
// InitialResettingTimerSliceCap is the initial slice capacity for the values
// stored in a ResettingTimer.

Reply:
Thank you - updated.